src/dispatch/static/dispatch/src/App.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-  <div class="app-root">
-    <router-view />
-    <v-snackbar bottom right :value="updateExists" :timeout="0" color="primary">
-      An update is available
-      <v-btn text @click="refreshApp">
-        Update
-      </v-btn>
-    </v-snackbar>
-  </div>
+  <v-app>
+    <div class="app-root">
+      <router-view />
+      <v-snackbar bottom right :value="updateExists" :timeout="0" color="info">
+        An update is available
+        <v-btn text @click="refreshApp">
```

Comment:
FYI When I made the changes to the snackbar component, I wasn't able to see the button unless I set the `color` property to something different than the one in the snackbar (e.g. `color="primary"`).